src/components/SidePanel/SidePanel.jsx

```diff
@@ -0,0 +1,207 @@
+import _ from 'lodash';
+import React from 'react';
+import PropTypes from 'react-peek/prop-types';
+import { lucidClassNames } from '../../util/style-helpers';
+import Overlay from '../Overlay/Overlay';
+import GripperVerticalIcon from '../Icon/GripperVerticalIcon/GripperVerticalIcon';
+import CrossIcon from '../Icon/CrossIcon/CrossIcon';
+import DragCaptureZone from '../DragCaptureZone/DragCaptureZone';
+import { getFirst, omitProps } from '../../util/component-types';
+
+const cx = lucidClassNames.bind('&-SidePanel');
+
+const { any, bool, func, oneOf, node, number, string } = PropTypes;
+
+class SidePanel extends React.Component {
+	constructor(...args) {
+		super(...args);
+		const [{ isExpanded, width }] = args;
+		this.state = {
+			isResizing: false,
+			width,
+			isExpanded,
+		};
+
+		this.handleResizeStart = this.handleResizeStart.bind(this);
```

Comment:
isn't this the old busted way to do this

Reply:
i just checked, we don't have babel configured to support that syntax yet, so i'll keep the bound methods for now and reconfigure babel in another PR